nodist/fuzzer/actual_fail

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+
+#Note: This assumes that this is Python 2.x
```

Comment:
Ignoring the issue of adding a new language to the project (Wilson can comment on that), we should at least start with Python 3.  Time is almost up on 2.7:
https://pythonclock.org/